package.json

```diff
@@ -17,39 +17,39 @@
     "axios": "0.19.2",
     "foundation-sites": "6.6.3",
     "vue": "^2.6.11",
-    "vue-good-table": "^2.19.3",
-    "vue-router": "^3.2.0",
-    "vuex": "^3.4.0"
+    "vue-good-table": "^2.19.5",
+    "vue-router": "^3.3.4",
+    "vuex": "^3.5.1"
   },
   "devDependencies": {
-    "@vue/cli-plugin-babel": "~4.4.0",
-    "@vue/cli-plugin-eslint": "~4.4.0",
-    "@vue/cli-plugin-router": "~4.4.0",
-    "@vue/cli-plugin-unit-jest": "~4.4.0",
-    "@vue/cli-plugin-vuex": "~4.4.0",
-    "@vue/cli-service": "~4.4.0",
+    "@vue/cli-plugin-babel": "~4.4.6",
+    "@vue/cli-plugin-eslint": "~4.4.6",
+    "@vue/cli-plugin-router": "~4.4.6",
+    "@vue/cli-plugin-unit-jest": "~4.4.6",
+    "@vue/cli-plugin-vuex": "~4.4.6",
+    "@vue/cli-service": "~4.4.6",
     "@vue/test-utils": "^1.0.3",
     "babel-eslint": "^10.1.0",
     "core-js": "^3.6.5",
-    "eslint": "^6.7.2",
+    "eslint": "^7.5.0",
     "eslint-config-tjw-base": "^1.0.0",
     "eslint-config-tjw-vue": "^2.0.0",
-    "eslint-plugin-jest": "^23.13.2",
+    "eslint-plugin-jest": "^23.18.0",
     "eslint-plugin-vue": "^6.2.2",
-    "eslint-plugin-vuejs-accessibility": "^0.1.3",
+    "eslint-plugin-vuejs-accessibility": "^0.3.1",
     "flush-promises": "^1.0.2",
-    "jest-environment-jsdom-global": "^2.0.2",
+    "jest-environment-jsdom-global": "^2.0.4",
     "jest-serializer-vue-tjw": "^3.14.0",
     "lodash.clonedeep": "^4.5.0",
     "normalize.css": "^8.0.1",
-    "sass": "^1.26.5",
+    "sass": "^1.26.10",
     "sass-lint": "^1.13.1",
-    "sass-loader": "^8.0.2",
+    "sass-loader": "^9.0.2",
     "tjw-sasslint-rules": "^2.1.0",
     "vue-template-compiler": "^2.6.11"
   },
   "engines": {
-    "node": ">=8.3.0",
+    "node": ">=13.7.0",
```

Comment:
When updating to `@vue/cli-service 4.4.6`, it updated it's own dependency of `colorette` which requires node 13.7+